### ISSUE TITLE
Bulgarian language support

### ISSUE THREAD

**Issue by tunnckoCore** [date-fns/date-fns#354]
i'll start it :bulgaria:

**Comment by LeshaKoss** [comment]
@tunnckoCore thank you! Looking forward to your contribution! 😄

**Comment by tunnckoCore** [comment]
Don't know but it's a bit hard to implement it :rofl: 

Almost built them as I think it should be, but it seems there's no "ordinal formatters" in the english variant, so when try to run the tests it fails 

```
1) en locale > buildFormatLocale formatters property Do returns ordinal result of D formatter:
     TypeError: formatters[formatterToken] is not a function
      at Object.formatters.(anonymous function) [as Do] (src/locale/bg/build_format_locale/index.js:70:48)
      at Context.<anonymous> (src/locale/bg/build_format_locale/test.js:316:47)
```

the part
```js
  var ordinalFormatters = ['M', 'D', 'DDD', 'd', 'Q', 'W']
  ordinalFormatters.forEach(function (formatterToken) {
    formatters[formatterToken + 'o'] = function (date, formatters) {
      return ordinal(formatters[formatterToken](date))
    }
  })
```

**Comment by tunnckoCore** [comment]
In that `var formatters` object, there's no formatter for `Mo`, but tests are passing. There's no formatter for `Do` (for example, `en` as base) too, but tests fails. Huh.

**Comment by arvigeus** [comment]
Not sure if this is correct #357 since it's my first pull request ever.

**Comment by leshakoss** [comment]
closed by #357
